client/src/scripts/managers/SceneManager.js

```diff
@@ -0,0 +1,101 @@
+import * as THREE from "three";
+import LabelsManager from "../LabelsManager";
+import Stats from "stats-js";
+import {OrbitControls} from "three/examples/jsm/controls/OrbitControls";
+
+export class SceneManager {
```

Comment:
Maybe named it screenInitializator or something liek that becouse it's not relly manager